### PR TITLE
allow use with react@0.13

### PR DESCRIPTION
**Pull request by iam4x**
address #16

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -34,7 +34,7 @@
     "webpack": "^1.4.13"
   },
   "peerDependencies": {
-    "react": "0.12.x"
+    "react": ">=0.12.0"
```
Comment (AsaAyers) [inline package.json]: Are you sure this is going to work on `0.14` when it gets released? Using the [semver calculator](http://semver.npmjs.com/) `^0.12 || ^0.13` will match 0.12 and 0.13.

Comment (iam4x) [inline package.json]: You are right!

**Comment by mathieumg** [comment]
:+1:

**Comment by wiredmax** [comment]
+1

**Comment by iam4x** [comment]
@mathieumg @wiredmax you can use `npm install --save-dev iam4x/react-a11y#packaged` as workaround until this is merged ;)

**Comment by bloodyowl** [comment]
:+1:

**Pull request closed without merging**